common.gradle

```diff
@@ -101,7 +101,7 @@ if (project.hasProperty("backend_monoscopic") && project.property("backend_monos
 }
 if (!hasBackend) {
     project.ext.backend_oculus = true
-    project.ext.backend_daydream = true
+    project.ext.backend_daydream = false
```

Comment:
Is this intentional?

Reply:
Hi @liaxim,
that is because of Dex issue mentioned at PR comments: 


Error:Execution failed for task ':gvr-arcore:transformDexArchiveWithExternalLibsDexMergerForDebug'.
> java.lang.RuntimeException: java.lang.RuntimeException: com.android.builder.dexing.DexArchiveMergerException: Unable to merge dex


That was my workaround but I still need help with this issue.